marlo_mcp/main.py

```
@@ -1,10 +1,11 @@
 from typing import Optional
 from uuid import UUID
+from typing import Optional
```

Ruff (0.12.2)

Redefinition of unused `Optional` from line 1

Remove definition: `Optional`

(F811)